### PR TITLE
fix ruff tests since last check

### PR DESCRIPTION
**Pull request by tylerflex**
since #1681

### CODE REVIEW

File: tests/test_plugins/test_component_modeler.py [hunk 1]
```diff
@@ -402,7 +390,7 @@ def test_import_smatrix_smatrix():
 
 def test_to_from_file_batch(monkeypatch, tmp_path):
     modeler = make_component_modeler(path_dir=str(tmp_path))
-    s_matrix = run_component_modeler(monkeypatch, modeler)
+    _ = run_component_modeler(monkeypatch, modeler)
```
Comment (yaugenst-flex) [inline tests/test_plugins/test_component_modeler.py:393]: Maybe no assignment at all?

File: tests/test_plugins/test_invdes.py [hunk 2]
```diff
@@ -72,10 +74,10 @@ def test_region_params():
 
     design_region = make_design_region()
 
-    PARAMS_0 = np.random.random(design_region.params_shape)
-    PARAMS_0 = design_region.params_random
-    PARAMS_0 = design_region.params_ones
-    PARAMS_0 = design_region.params_zeros
+    _ = np.random.random(design_region.params_shape)
```
Comment (yaugenst-flex) [inline tests/test_plugins/test_invdes.py:77]: Same here, since it's just testing whether the properties error.

**Review by yaugenst-flex — COMMENTED** [review]
Nice thanks. Guess the only question is whether things that are not used should be assigned at all. I went with no assignment in my last fixes, here it's `_ =`.

File: tests/test_plugins/test_component_modeler.py (hunk 1 above)
Comment (tylerflex) [reply tests/test_plugins/test_component_modeler.py:393]: hm, doesn't really bother me. If we want to extend the tests later, it's kind of nice to know that *something* is returned there. In fact, I dont mind even leaving it as a variable..

Comment (yaugenst-flex) [reply tests/test_plugins/test_component_modeler.py:393]: OK let's do it like this then. I think we can merge this now and figure out a linting config that makes sense for the tests in #1680

**Pull request merged**